l10n_nl_postcodeapi/models/res_partner.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2020 Therp BV <https://therp.nl>
+# @autors: Stefan Rijnhart, Ronald Portier
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.model
+    def get_provider_obj(self):
+        apikey = self.env['ir.config_parameter'].sudo().get_param(
+            'l10n_nl_postcodeapi.apikey', '').strip()
+        if not apikey or apikey == 'Your API key':
+            return False
+        from pyPostcode import Api
```

Comment:
Can you move this importing statement `from pyPostcode import Api` to the top of the file?